src/compass/s1_geocode_slc.py

```diff
@@ -247,8 +247,7 @@ def run(cfg: GeoRunConfig):
         t_qa_meta = time.perf_counter()
         with h5py.File(output_hdf5, 'a') as geo_burst_h5:
             root_group = geo_burst_h5[ROOT_PATH]
-            identity_to_h5group(root_group, burst, cfg, 'CSLC -S1',
-                                cfg.product_group.product_specification_version)
+            identity_to_h5group(root_group, burst, cfg, 'CSLC -S1')
```

Comment:
Is there supposed to be a space between `CSLC` and `-S1`?

Reply:
No, I goofed. This should have been fixed by the latest commit